agentstack/proj_templates.py

```diff
@@ -124,8 +171,10 @@ def from_json(cls, data: dict) -> 'TemplateConfig':
         try:
             match data.get('template_version'):
                 case 1:
-                    return TemplateConfig_v1(**data).to_v2()
+                    return TemplateConfig_v1(**data).to_v2().to_v3()
```

Comment:
im not a huge fan of this moving forward, but for now i get it

Reply:
Yeah, not advocating we keep stacking like that indefinitely, but it does keep it DRY for now.